lib/tasks/gettext.rake

```diff
@@ -1,21 +1,23 @@
 # -*- encoding : utf-8 -*-
 namespace :gettext do
-  def msgmerge
-    Rails.application.config.gettext_i18n_rails.msgmerge ||
-      %w[--sort-output --no-location --no-wrap]
+  def msgmerge(*files)
+    destination = files.first
+    options = %w[--sort-output --no-location --no-wrap --no-obsolete-entries]
```

Comment:
Added `no-obsolete-entries` to prevent warnings in STDOUT

Reply:
👍  – squash into commit message

Reply:
might as well merge `--output` flag into these now?

Reply:
We could but the `--output` flag requires a value. So the command is evaluated to something like:
`msgmerge *our options* --output=$(output.path) *our files*` but without calling the command because its all done in Ruby code.

I intentional left it out so it can be next to the actual output path.

Reply:
Ah! Okay yeah makes more sense as is 👍